spec/lib/tasks/womenrising/peer_group_monthly_matches_rake_spec.rb

```diff
@@ -1,29 +1,21 @@
 require "rails_helper"
 
-describe "womenrising:peer_group_monthly_match" do
+describe "womenrising:peer_group_signup_reminder" do
```

Comment:
It looks like the tasks got mixed up based on the names of the files

Reply:
Thanks for catching that, and for adding the screenshot.